src/PrestaShopBundle/Resources/views/Admin/Translations/include/translations-tree.html.twig

```diff
@@ -0,0 +1,13 @@
+<div class="domains-tree col-md-4">
+  <a class="btn-expand">
+    <i class="material-icons">&#xE315;</i>
+    <span>{{ 'Expand all'|trans({}, 'Admin.International.Feature') }}</span>
+  </a>
+  <a class="btn-reduce">
+    <i class="material-icons">&#xE315;</i>
+    <span>{{ 'Reduce all'|trans({}, 'Admin.International.Feature') }}</span>
```

Comment:
My mistake I gave the wrong wording, we use "Collapse all" in other pages.
Can you please change and use "Admin.Actions"? Thanks.